meseq.py

```diff
@@ -176,8 +178,14 @@ def text(self, x, y, text, flags = ALIGN_CENTER):
         layout.set_alignment(pango.ALIGN_CENTER)
         w, h = layout.get_size()
         w = w / pango.SCALE
-        h = h / pango.SCALE
+        h = h / pango.SCALE + 1
+        if white_bg:
```

Comment:
Could you please add a new example to illustrate this feature? With the existing examples, I do not see the benefit, and on the contrary some arrows are slightly damaged.